.pre-commit-hooks.yml

```diff
@@ -0,0 +1,7 @@
+- id: infosec-precommit
```

Comment:
It seems this step is unnecessary. This is for creating new hooks, see https://pre-commit.com/#creating-new-hooks. 

However, to use existing hooks like [infosec-precommit](https://github.com/elastic/infosec-precommit), you only need a `.pre-commit-config.yaml`, as shown in https://pre-commit.com/#adding-pre-commit-plugins-to-your-project.